### PR TITLE
LPS-194126 | Show map string object localized fields in fragment

### PR DESCRIPTION
**Pull request by andreluis2022**
**Description:**
Create tests - Product QA | Functional Automation - LPS-194126 Show map string object localized fields in fragment

**JIRA Ticket:**
https://liferay.atlassian.net/browse/LPS-194126

### CODE REVIEW

**Comment by andreluis2022** [comment]
ci:test:frontend-data-set

**Comment by andreluis2022** [comment]
ci:test:sf

**Comment by andreluis2022** [comment]
ci:test:frontend-data-set

**Comment by andreluis2022** [comment]
Hey @CarlosBrichete can you take a look ?

**Review by CarlosBrichete — APPROVED** [review]
Check my change requests

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSet.testcase [hunk 1]
```diff
@@ -1244,6 +1414,239 @@ definition {
...
+		task ("And go to the page created, and put a "pt" in URL page") {
+			Navigator.openURL();
+
+			Navigator.gotoPage(pageName = "Data Set Test");
+
+			Navigator.openSpecificURL(url = "http://localhost:8080/pt/data-set-test");
+
+			AssertElementPresent(
+				key_column = "Nome",
+				locator1 = "ObjectPortlet#ENTRY_COLUMN");
```
[the remark this answers is not in the document]
Comment (andreluis2022) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSet.testcase]: All done!

**Comment by andreluis2022** [comment]
ci:test:sf

**Comment by andreluis2022** [comment]
Hey @CarlosBrichete, what do you think?

**Comment by andreluis2022** [comment]
ci:test:frontend

**Review by CarlosBrichete — CHANGES_REQUESTED** [review]
Fix the task description and then you can forward :) No need to run the tests again if they are already passing, just run the SF and then forward it

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSet.testcase [hunk 2]
```diff
@@ -1152,6 +1236,92 @@ definition {
 		}
 	}
 
+	@description = "LPS-194126 Confirm that when creating an object field with translation, a new view fields value is not displayed"
+	@priority = 3
+	test NewFieldValueIsNotDisplayed {
+		task ("Givee") {
```
[the remark this answers is not in the document]
Comment (andreluis2022) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSet.testcase:1242]: Ohh sorry

**Comment by andreluis2022** [comment]
Done!

**Review by CarlosBrichete — APPROVED** [review]

**Comment by andreluis2022** [comment]
ci:forward

**Pull request closed without merging**